enginetest/script_queries.go

```diff
@@ -320,4 +320,40 @@ var ScriptTests = []ScriptTest{
 			},
 		},
 	},
+	{
+		Name: "UUIDs used in the wild.",
+		SetUpScript: []string{
+			"SET @uuid = '6ccd780c-baba-1026-9564-5b8c656024db'",
+		},
+		Assertions: []ScriptTestAssertion{
+			{
+				Query:    `SELECT IS_UUID(@uuid)`,
+				Expected: []sql.Row{{int8(1)}},
+			},
+			{
+				Query:    `SELECT BIN_TO_UUID(UUID_TO_BIN(@uuid))`,
```

Comment:
We should add a test for accepting byte arrays as well. In MySQL this is valid:
`SELECT BIN_TO_UUID(X'00112233445566778899aabbccddeeff');`
But we don't have a way to represent those directly, so a byte array would work as a substitution.

Also, we need raw string input.
`SELECT BIN_TO_UUID('0011223344556677');` is also valid, which we would just take the string bytes as a byte array.